code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -84,7 +84,7 @@
 	name = "pulse destroyer"
 	desc = "A heavy-duty energy rifle built for pure destruction."
 	worn_icon_state = "pulse"
-	cell_type = "/obj/item/stock_parts/cell/infinite"
+	cell_type = /obj/item/stock_parts/cell/infinite
```

Comment:
This power cell isn't actually infinite and can be removed like the other snowflake cells. Set the power to something completely ridiculous like 50 gigawatts